packages/eslint-plugin/tsconfig.build.json

```diff
@@ -2,8 +2,8 @@
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     // specifically disable declarations for the plugin
```

Comment:
Can you remove this comment as it’s no longer accurate?

Reply:
Actually, removing the two `declaration` options will cause a fallback to the inherited config, which enables them.